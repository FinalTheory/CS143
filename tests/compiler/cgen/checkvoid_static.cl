class Foo {
  f() : Object {
    (new IO).out_string("Hi from Foo_f\n")
  };
};

class Main {
  f : Foo;
  main() : Object {
    f@Foo.f()
  };
};
