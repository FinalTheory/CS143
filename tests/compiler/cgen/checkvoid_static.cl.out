SPIM Version 6.5 of January 4, 2003
Copyright 1990-2003 by James R. Larus (larus@cs.wisc.edu).
All Rights Reserved.
See the file README for a full copyright notice.
Loaded: /usr/class/cs143/cool/lib/trap.handler
checkvoid_static.cl:1: Dispatch to void.
Stats -- #instructions : 138
         #reads : 20  #writes 19  #branches 25  #other 74
